.github/workflows/build_msquic_openssl.yml

```diff
@@ -0,0 +1,34 @@
+name: "Build msquic with openssl interop container"
```

Comment:
Why did you create a new workflow instead of adding a new job into https://github.com/openssl/openssl/blob/master/.github/workflows/build_quic_interop_container.yml